pom.xml

```diff
@@ -121,6 +121,7 @@
         <module>znai-shaded</module>
         <module>znai-dist</module>
         <module>znai-tests</module>
+        <module>znai-mermaid</module>
```

Comment:
let's add it to existing `diagrams` module

Reply:
do you mean add it as a dependency of `diagrams` or move the code entirely there?

Reply:
move the code under diagrams maven module